android/src/com/mapswithme/maps/editor/EditorHostFragment.java

```diff
@@ -48,29 +49,31 @@
   /**
    * A list of localized names added by a user and those that were in metadata.
    */
-  private static final List<LocalizedName> sLocalizedNames = new ArrayList<>();
+  private static final List<LocalizedName> sNames = new ArrayList<>();
+  /**
+   * Count of names which should always be shown.
+   */
+  private int mMandatoryNamesCount = 0;
 
   /**
    *   Used in MultilanguageAdapter to show, select and remove items.
    */
-  List<LocalizedName> getLocalizedNames()
+  List<LocalizedName> getNames()
   {
-    return sLocalizedNames;
+    return sNames;
   }
 
-  public LocalizedName[] getLocalizedNamesAsArray()
+  public LocalizedName[] getNamesAsArray()
   {
-    return sLocalizedNames.toArray(new LocalizedName[sLocalizedNames.size()]);
+    return sNames.toArray(new LocalizedName[sNames.size()]);
   }
 
-  void setLocalizedNames(LocalizedName[] names)
+  void setNames(LocalizedName[] names)
   {
-    sLocalizedNames.clear();
+    sNames.clear();
     for (LocalizedName name : names)
     {
```

Comment:
Up to you, but feel free to get rid of curly braces, because loop body is a one-liner.